drivers/storage/portworx/component/plugin.go

```diff
@@ -354,3 +360,23 @@ func isVersionSupported(v string) bool {
 
 	return currentVersion.GreaterThanOrEqual(targetVersion)
 }
+
+func getDesiredPluginImage(cluster *corev1.StorageCluster) string {
+	imageName := "portworx/portworx-dynamic-plugin:1.0.0"
```

Comment:
Let's please move the `portworx/portworx-dynamic-plugin:1.0.0` as one of the constants -- ideally together w/ other image-constants in drivers/storage/portworx/manifest/manifest.go ?

Reply:
Additionally, the manifest should be adding the default values and returning them. Just follow the pattern that other component images use. Let's assume that the desired image is populated here.

Reply:
done